docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.9'
+
+services:
+  bot:
+    build: .
+    image: white-whale-bot
+    container_name: white-whale-bot
+    restart: always
+    stdin_open: true
+    tty: true
```

Comment:
can we add the environment file in here?

Reply:
Yes, but docker-compose limits env variables to a single line so some changes in how pools are read need to happen first. Ideally, we'd just read in the env variables here so they could be altered on a per container basis and not copied into the docker image.  At that point, y'all could publish a docker image on docker hub and bots could be spun up without the need to download src.